docs/frost-urlinput.md

```diff
@@ -0,0 +1,163 @@
+# frost-text
```

Comment:
It looks like this file has lots of references to `frost-text` that should actually be `frost-urlinput`.